### ISSUE TITLE
[boost] boost/1.72.0: Not able to build due to Forbidden!

### ISSUE THREAD

**Issue by datalogics-kam** [conan-io/conan-center-index#1219]
The basic problem is that Boost.org is over their OSS limit, see boostorg/boost#383. 

Would it be possible to point source downloads at a more reliable source?

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **boost/1.72.0**
  * Operating System+version: n/a
  * Compiler+version: n/a
  * Docker image: n/a
  * Conan version: n/a
  * Python version: n/a

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
- happens with any profile (we have many CI builds that are at issue)

### Steps to reproduce (Include if Applicable)

Do any build that requires downloading source for `boost/1.72.0`

### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
conans.errors.ConanExceptionInUserConanfileMethod: boost/1.72.0: Error in source() method, line 151
	tools.get(**self.conan_data["sources"][self.version])
	AuthenticationException: Forbidden!
```

</details>

**Comment by Croydon** [comment]
Maybe we should switch the download to GitHub release downloads? https://github.com/boostorg/boost/releases

**Comment by uilianries** [comment]
Yes, we can just put as mirror.

**Comment by danimtb** [comment]
I think it was fixed already. Please check

**Comment by datalogics-kam** [comment]
It looks like the Bintray access came back. Thanks for looking at it.

The mirror idea sounds good.

**Comment by danimtb** [comment]
@uilianries @Croydon I am closing this issue but please if you think adding the mirror is something we should do, open a new issue for that.  Thanks!

**Comment by uilianries** [comment]
Sure! Now with Conan 1.24 we add any mirror easily.

**Comment by wouterbeek** [comment]
Today I am having this issue as well.

@danimtb Is it still an option to add the mirror that you mentioned earlier?

**Comment by erikzenker** [comment]
Also had the issue today. So I guess no mirror was added?

The issue is resolved always when the download limit is reset every month.

**Comment by aaaaaaaaargh** [comment]
Also affects **1.73.0**

Luckily I was smart enough to look here first before I ripped my CI apart. Is there any explanation on why this is happening and, more importantly, when this will be fixed?

**Comment by erikzenker** [comment]
@aaaaaaaaargh it happens because the download limit of the boost org webpage is reached. You can reproduce it by downloading boost manually. It could be work around by adding github as mirror to the package.

**Comment by aaaaaaaaargh** [comment]
Oh, I see. So the cause of it isn't conan then. I'm not too experienced with packages, so I don't know how complicated it is to add github as a mirror. My guess would be that it's not too hard probably, but I really don't know.

**Comment by aaaaaaaaargh** [comment]
Just a quick update on this: I tried to modify the recipe locally with links to the same file hosted on sourceforce. This seems to work pretty well, although I don't know whether this would be an option. Anyways, I strongly suggest to change the download location of this package as now we will probably have to wait until 1th of July again before the traffic will be reset.

For further reference, please see the corresponding discussion on the boost mailing ist:
https://lists.boost.org/boost-users/2020/06/90495.php

**Comment by erikzenker** [comment]
> Also, is there anything we can do to circumvent the problem in the meantime?

For now, I will clone the package into my own bintray remote and change the package source. At least this will unblock my CI. But in general the "official conan" package should be made fault tolerant to such situations for the future.

**Comment by danimtb** [comment]
See #1860

**Comment by erikzenker** [comment]
The package was updated and the issue is resolved for me

**Comment by aaaaaaaaargh** [comment]
Yes, looks good to me too.
